pkg/email/trb_advice_letter_submitted.go

```diff
@@ -0,0 +1,86 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"path"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// SendTRBAdviceLetterSubmittedEmailInput contains the data needed to to send the TRB advice
+// letter submitted email
+type SendTRBAdviceLetterSubmittedEmailInput struct {
+	TRBRequestID   uuid.UUID
+	RequestName    string
+	RequestType    string
+	RequesterName  string
+	Component      string
+	SubmissionDate *time.Time
+	ConsultDate    *time.Time
+	Recipients     []models.EmailAddress
+}
+
+// trbAdviceLetterSubmittedEmailTemplateParams contains the data needed for interpolation in
+// the TRB advice letter submitted email template
+type trbAdviceLetterSubmittedEmailTemplateParams struct {
+	RequestName         string
+	RequesterName       string
+	Component           string
+	RequestType         string
+	SubmissionDate      string
+	ConsultDate         string
+	TRBAdviceLetterLink string
+	TRBRequestLink      string
+	TRBInboxAddress     string
+	TRBEmail            models.EmailAddress
+}
+
+// SendTRBAdviceLetterSubmittedEmail sends an email to the EASI admin team indicating that an advice letter
+// has been submitted
+func (c Client) SendTRBAdviceLetterSubmittedEmail(ctx context.Context, input SendTRBAdviceLetterSubmittedEmailInput) error {
+	subject := "Advice letter added for " + input.RequestName
+
+	allRecipients := append(input.Recipients, c.config.TRBEmail)
+
+	submissionDate := ""
+	if input.SubmissionDate != nil {
+		submissionDate = input.SubmissionDate.Format("January 2, 2006")
+	}
```

Comment:
Is it possible to have an advice letter for something that hasn't been submitted? I suppose it might technically be possible, but I imagine generally won't happen. We might not need to do anything here, but wanted to make sure I understood why this block was needed